py-json/interop_connectivity.py

```diff
@@ -533,33 +1025,76 @@ async def set_port(self, port_list=[]):
             resource = port_data['resource']
             port = port_data['sta_name']
             interest = port_data['interest']
-
+            # report_timer = port_data['report_timer']
+            current_flags = port_data['current_flags']
             os = port_data['os']
-            if (os in ['Apple', 'Lin']):
-                current_flags = port_data['current_flags']
+            if (os == 'Lin'):
                 data = {
                     'shelf': shelf,
                     'resource': resource,
                     'port': port,
-                    'current_flags': current_flags,
-                    'interest': interest,
+                    "report_timer": 1,
+                    'current_flags': 2147483648,
+                    'interest': 92291074,
                     'mac': self.mac
                 }
-            elif (os == 'Win'):
-                report_timer = port_data['report_timer']
-                current_flags = port_data['current_flags']
+            else:
                 data = {
                     'shelf': shelf,
                     'resource': resource,
                     'port': port,
-                    'report_timer': report_timer,
-                    'current_flags': current_flags,
-                    'interest': interest
+                    "report_timer": 1,
+                    'current_flags': 2147483648,
+                    'interest': 92291074,
```

Comment:
Same here, please comment current and interest flags. Not required for review, but generally best not to mix quotes stylistically speaking (nit of `report_timer`)